.github/workflows/ci.yml

```diff
@@ -0,0 +1,187 @@
+    # If you do not wish to use codecov, then simply delete this job from the
+    # workflow.
+    if: github.repository_owner == 'beached'
+      && false
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install LCov
+      run: sudo apt-get update -q
+        && sudo apt-get install lcov -q -y
+
+    - name: Install vcpkg
+      uses: lukka/run-vcpkg@v7
+      with:
+        setupOnly: true
+        vcpkgGitCommitId: ${{ env.VCPKG_COMMIT }}
+
+    - name: Configure
+      env: { CXX: g++-10 }
+      run: cmake --preset=ci-coverage -D CMAKE_CXX_STANDARD=${{ matrix.std }}
+        -D DAW_NO_FLATTEN=YES
+
+    - name: Build
+      run: cmake --build build/coverage -j 2
+
+    - name: Test
+      working-directory: build/coverage
+      run: ctest --output-on-failure -j 2
+
+    - name: Process coverage info
+      run: cmake --build build/coverage -t coverage
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: coverage-${{ matrix.std }}
+        path: build/coverage/coverage.info
+        retention-days: 5
+
+  codecov:
+    needs: [coverage]
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install LCov
+      run: sudo apt-get update -q
+        && sudo apt-get install lcov -q -y
+
+    - uses: actions/download-artifact@v2
+      with: 
+        name: coverage-17
+        path: "17"
+
+    - uses: actions/download-artifact@v2
+      with: 
+        name: coverage-20
+        path: "20"
+
+    - name: Combine coverage reports
+      run: lcov -o coverage.info -a 17/coverage.info -a 20/coverage.info
+
+    - name: Submit to codecov.io
+      uses: codecov/codecov-action@v1
+      with:
+        file: coverage.info
+
+  sanitize:
+    needs: [spell-check]
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        std: [17, 20]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install vcpkg
+      uses: lukka/run-vcpkg@v7
+      with:
+        setupOnly: true
+        vcpkgGitCommitId: ${{ env.VCPKG_COMMIT }}
+
+    - name: Configure
+      env: { CXX: clang++-12 }
+      run: cmake --preset=ci-sanitize -D CMAKE_CXX_STANDARD=${{ matrix.std }}
+
+    - name: Build
+      run: cmake --build build/sanitize -j 2
+
+    - name: Test
+      working-directory: build/sanitize
+      env:
+        ASAN_OPTIONS: "strict_string_checks=1:\
+          detect_stack_use_after_return=1:\
+          check_initialization_order=1:\
+          strict_init_order=1:\
+          detect_leaks=1"
+        UBSAN_OPTIONS: print_stacktrace=1
+      run: ctest --output-on-failure -LE excluded -j 2
+
+  test:
+    needs: [spell-check]
+
+    strategy:
+      matrix:
+        os: [macos, ubuntu, windows]
+
+        std: [17, 20]
+
+        include:
+        - os: ubuntu
+          env: '{ "CXX": "g++-10" }'
+          flags: -D DAW_NO_FLATTEN=YES
+
+    runs-on: ${{ matrix.os }}-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install vcpkg
+      uses: lukka/run-vcpkg@v7
```

Comment:
This is MIT licensed and incompatible with BSL.  Still working my way down, but I don't understand what this does.  The two dependencies are super small and header only.

Reply:
What does `this` refer to here? vcpkg? The `run-vcpkg` action? You were already using nanobench, which is also MIT license. License incompatibilities only matter for distribution and runtime usage generally.

The purpose of this action is to set vcpkg and the necessary env variables up at the given commit conveniently.